spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/pubsub/GcpPubSubProperties.java

```diff
@@ -43,6 +47,8 @@
 	 */
 	private String emulatorHost;
 
+	private List<String> trustedPackages = Collections.emptyList();
```

Comment:
Add some Javadoc here?
I'd argue that `jsonMapperTrustedPackages` would be a more elucidating name.

Reply:
Well, there is similar deserialization vulnerabilities with the regular Java `Serializable`.
When we introduce such a converter we should ensure to use similar property as well.
So, I think to narrow the name to the `json` would eliminate a possibility to reuse the property in other places based on the trusted sources.

So, my point is to leave the name as is.

Reply:
Then let's change it to `deserializationTrustedPackages`.

Anything more informative than `trustedPackages`. Trusted packages of what? For what? This was my initial reaction when I saw this field.

Reply:
Right. That's why we always encourage to add JavaDocs to this kind of configuration properties.